presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -268,6 +268,21 @@ else if (varcharType.getBoundedLength() <= 16777215) {
         return super.toWriteMapping(session, type);
     }
 
+    private boolean isGTIDMode(Connection connection)
+    {
+        try (java.sql.Statement statement = connection.createStatement()) {
+            ResultSet resultSet = statement.executeQuery("show variables like 'gtid_mode'");
```

Comment:
We can make this line try-with-resources and please upper case for keywords. 
```suggestion
            ResultSet resultSet = statement.executeQuery("SHOW VARIABLES LIKE 'gtid_mode'");
```